ground/src/main/java/com/google/android/ground/persistence/sync/MediaUploadWorker.kt

```diff
@@ -133,23 +94,10 @@ constructor(
         throw FileNotFoundException(photoFile.path)
       }
       remoteStorageManager.uploadMediaFromFile(photoFile, path)
-      kotlin.Result.success(Unit)
-    } catch (t: FirebaseNetworkException) {
-      Timber.d(t, "Can't connect to Firebase to upload photo")
-      kotlin.Result.failure(t)
+      return kotlin.Result.success(Unit)
     } catch (t: Throwable) {
-      Timber.e(t, "Failed to upload photo")
-      kotlin.Result.failure(t)
+      Timber.log(t.priority(), t, "Photo upload failed")
```

Comment:
Please see the other comment regarding reverting this change

Reply:
Forked this discussion into a suggestion in #2891.